lib/gem_ext/rails/reflection.rb

```diff
@@ -0,0 +1,59 @@
+require 'active_record/reflection'
+
+ActiveRecord::Reflection.module_eval do
+  def self.create(macro, name, scope, options, ar)
```

Comment:
I'm not sure. I'm going to extract it into a gem for sure, but I don't know if they'd want it in Rails since it'll only work with Postgres.